lib/filter-headers.js

```diff
@@ -1,23 +1,41 @@
 'use strict';
-const ACCEPT_HEADERS = ['accept-language', 'referer', 'user-agent'];
-/**
- * Filter the request headers that are passed to fragment request.
- * @callback filterHeaders
- *
- * @param {Object} attributes - Attributes object of the fragment node
- * @param {string} attributes.public - Denotes the public fragment.
- * @param {Object} request - HTTP Request object
- * @param {Object} request.headers - request header object
- * @returns {Object} New filtered header object
- */
-module.exports = (attributes, request) => {
-    const { public: publicFragment } = attributes;
-    const { headers = {} } = request;
-    // Headers are not forwarded to public fragment for security reasons
+const ACCEPT_REQ_HEADERS = ['accept-language', 'referer', 'user-agent'];
+const ACCEPT_RES_HEADERS = ['location', 'set-cookie'];
```

Comment:
passing set-cookie as default will break if people use reverse proxy infront of tailor and set the cookie from there. Keep only location for now. others can override this function if required.